@navikt/core/react/src/form-summary/FormSummaryHeader.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { Box } from "../layout/box";
+import { HStack } from "../layout/stack";
+
+interface FormSummaryHeaderProps {
+  children: React.ReactNode;
+}
+
+export default function FormSummaryHeader({
+  children,
+}: FormSummaryHeaderProps) {
+  return (
+    <Box
+      as="header"
+      background="surface-subtle"
+      borderRadius="large large 0 0"
+      borderWidth="0 0 1 0"
+      borderColor="border-subtle"
+      paddingBlock="4"
+      paddingInline="6"
```

Comment:
Litt vanskeligere å trace hvor og hva som setter padding etc nå siden noe av paddingen defineres i form-summary.css og noe gjøre inline med Box. Bør alt defineres likt for komponenten?

Reply:
Box er ikke bra nok / generisk nok (dette er jo et problem som consumers av Box kommer borti hele tiden 🤔 ).

1. fjerne Box (do it all in classnames)
  - bra for DOM nodes (blir en ekstra div ellers?) 
3. beholde Box (bare layout i Box + annet som ikke er layout i classnames)?

Reply:
Har lurt litt på om Box kunne/burde hatt støtte for flex, men da blir det jo seff litt overlapp med Stack... 🤔

Dette funker 🤪
```tsx
  <Box
    ref={ref}
    as={(props) => (
      <HStack
        as="header"
        justify="space-between"
        gap="2"
        wrap={false}
        {...props}
      />
    )}
    background="surface-subtle"
    borderRadius="large large 0 0"
    borderWidth="0 0 1 0"
    borderColor="border-subtle"
    paddingBlock="4"
    paddingInline="6"
    {...rest}
  >
    {children}
  </Box>
```

Men uansett ser jeg poenget med å ha stylingen definert på en konsekvent måte.

Reply:
Den er funky 😂 ... render prop i `as`. Have Layout components gone _too_ far?